components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -270,6 +270,9 @@ GreaselionServiceImpl::GetExtensionIdsForTesting() {
 }
 
 void GreaselionServiceImpl::UpdateInstalledExtensions() {
+  if (!delegate_->IsEnabled()) {
+    return;
```

Comment:
Will this unload currently installed Greaselion extensions if it's disabled? If not, should it? I'm wondering if this check could go in `CreateAndInstallExtensions`. @emerick WDYT?

Reply:
Makes sense to me.